cpuinfo_test.go

```diff
@@ -402,3 +466,16 @@ func TestCPUInfoParseRISCV64(t *testing.T) {
 		t.Errorf("want ModelName %v, have %v", want, have)
 	}
 }
+
+func TestCPUInfoParseLOONG64(t *testing.T) {
+	cpuinfo, err := parseCPUInfoLoong([]byte(cpuinfoLoong64))
+	if err != nil || cpuinfo == nil {
+		t.Fatalf("unable to parse loongarch cpu info: %v", err)
```

Comment:
```suggestion
		t.Fatalf("unable to parse loong cpu info: %v", err)
```

Reply:
Thank you for reviewing, I have modified the code